### PR TITLE
Change api to `~set.constant`; fixes #3

### LINKED ISSUE #3: Long syntax
Syntax is too long. Why not just `~color.primary`?

### PR DESCRIPTION
**Pull request by ojame**
Changes the api:

`color: blue from ~colors;`

:point_down: 

`color: ~colors.blue;`

### CODE REVIEW

**Comment by ticky** [comment]
This seems like a big move given how much we’re using the old one. Would it be that much extra effort to support both?

**Comment by nelix** [comment]
Should be a pretty easy upgrade, best to get rid of the old ugly syntax sooner rather than later.

**Comment by ticky** [comment]
Fair enough. This is missing an update to the docs, BTW!

**Comment by ojame** [comment]
Heh, good pickup!

**Comment by ojame** [comment]
Fixed readme

File: index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = postcss.plugin('postcss-local-constants', function (opts) {
     var sets = opts && opts.defaults || {};
     var globalNode;
 
-    var regex = /((?:[\w]+))( )(from)(\s+)(~)((?:[A-z]+))/g;
+    var regex = /(~)((?:[\w]+))(\.)((?:[\w]+))/g;
```
Comment (ticky) [inline index.js]: Having a look at this there seem to be a lot of redundant capture groups we’re not using?

I feel like `~([\w]+)\.([\w]+)` would do the same job, and allow the match indexes to just be `2` (for `constant`) and `1` (`constantSet`).

**Comment by ticky** [comment]
I’m not sure who has sign-off approval on this :sweat_smile:

**Pull request merged**